client/src/components/AllTabs/AllTabs.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./AllTabs.css";
+import { useState } from "react";
+import PainGraph from "../DistressGraphs/PainGraph";
+import BowelsGraph from "../DistressGraphs/BowelsGraph";
+import BreathingGraph from "../DistressGraphs/BreathingGraph";
+import AppetiteGraph from "../DistressGraphs/AppetiteGraph";
+import NauseaGraph from "../DistressGraphs/NauseaGraph";
+import FatigueGraph from "../DistressGraphs/FatigueGraph";
+import SleepingGraph from "../DistressGraphs/SleepingGraph";
+import VolumeGraph from "../VolumeGraph/VolumeGraph";
+
+function AllTabs({dataLogs}) {
+  const [activeTab, setActiveTab] = useState("tab1");
+  function handleTabClick(tab) {
+    setActiveTab(tab);
+  }
+  return (
+    <div className="Tabs">
+      <ul className="nav">
+        <li
+          className={activeTab === "tab1" ? "active" : ""}
```

Comment:
Advice - look up what an enum is. This is a clean up task but I would replace tabX with a specific enum. 

Another way you can do this is have a list and write a for loop over this list to generate each li element. This way as you scale up the number of tabs, it doesn't trickle over all this extra code.